services/github/github-package-lock-json.service.js

```diff
@@ -0,0 +1,281 @@
+'use strict'
+
+const { renderVersionBadge } = require('../version')
+const {
+  transformAndValidate,
+  renderDynamicBadge,
+} = require('../dynamic-common')
+const {
+  isPackageLockJsonWithDependencies,
+  getLockDependencyVersion,
+} = require('../package-lock-json-helpers')
+const { semver } = require('../validators')
+const { ConditionalGithubAuthV3Service } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+const Joi = require('@hapi/joi')
+
+const keywords = ['npm', 'node']
+
+const versionSchema = Joi.object({
+  version: semver,
+}).required()
+
+// Leave this for completion's sake.
+// Usually you'll go ahead and get the version from package.json instead.
+class GithubPackageLockJsonVersion extends ConditionalGithubAuthV3Service {
```

Comment:
It seems like this badge is explicitly to provide the `version` key value from the package lockfile of the package itself (not dependnecies). What's the use case or need for this? It seems duplicative of existing functionality Shields already provides.

Reply:
I added it for completeness, there isn't any significant use case I can think of.

Reply:
Thanks! Let's remove it then, we don't want to have to maintain code and tests unnecessarily, and removing it should simplify this file a bit too